packages/build/webpack/webpack.base.js

```diff
@@ -37,20 +47,30 @@ const configFactory = {
     },
   },
   rules: {
+    raw() {
+      return {
+        resourceQuery: /raw/,
+        type: 'asset/source',
+      };
+    },
     fonts() {
       return {
         test: /fonts\/(.)+\.(woff|woff2|ttf|eot|svg)/,
-        loader: 'url-loader',
-        options: {
-          limit: 102400, // 100kb
-          name: '/assets/fonts/[name].[ext]',
+        type: 'asset',
+        generator: {
+          filename: 'assets/fonts/[name][ext]',
```

Comment:
I assume the `.` is now part of the extension?

Reply:
Yeah - https://webpack.js.org/guides/asset-modules/#custom-output-filename